apypie/foreman.py

```diff
@@ -0,0 +1,228 @@
+"""
+Apypie Foreman module
+
+opinionated helpers to use Apypie with Foreman
+"""
+import time
+
+try:
+    from typing import cast, Optional, Set, Tuple
+except ImportError:
+    pass
+
+from apypie.api import Api
+
+from apypie.resource import Resource  # pylint: disable=unused-import
+
+PER_PAGE = 2 << 31
```

Comment:
I'd appreciate a comment that describes the `per_page=all` vs Katello's solution and that we use this hack because it's universal

Reply:
added